core/gui/src/app/workspace/component/file-selection/file-selection.component.ts

```diff
@@ -16,15 +16,11 @@ import {
 })
 export class FileSelectionComponent {
   readonly fileTreeNodes: ReadonlyArray<DatasetVersionFileTreeNode> = inject(NZ_MODAL_DATA).fileTreeNodes;
-  suggestedFileTreeNodes: DatasetVersionFileTreeNode[] = [];
+  suggestedFileTreeNodes: DatasetVersionFileTreeNode[] = inject(NZ_MODAL_DATA).fileTreeNodes;
```

Comment:
`suggestedFileTreeNodes: DatasetVersionFileTreeNode[] = this.fileTreeNodes` might be a better practice when assigning two variables to the same value.

Reply:
TS4104: The type  readonly DatasetVersionFileTreeNode[]  is  readonly  and cannot be assigned to the mutable type  DatasetVersionFileTreeNode[] 

I am getting this error by writing this way.

Reply:
Maybe I will go with adding the `NgOnInit` back

Reply:
You can do it this way: `suggestedFileTreeNodes: DatasetVersionFileTreeNode[] = [...this.fileTreeNodes];`